test/integration/tests/allow-newer-specific-packages/files/stack.yaml

```diff
@@ -0,0 +1,4 @@
+resolver: nightly-2022-10-18
```

Comment:
The integration tests are currently standardised around `nightly-2020-09-05`. It makes them easier to update, if they all remain in line with each other.

Reply:
Fixed, thank you